manifests/service.pp

```diff
@@ -27,16 +12,4 @@
       hasrestart => true,
     }
   )
-
-  if $mkhomedir and $manage_oddjobd {
-    ensure_resource('service', 'oddjobd',
-      {
-        ensure     => $service_ensure,
-        enable     => true,
-        hasstatus  => true,
-        hasrestart => true,
-        require    => Service[$sssd_service],
```

Comment:
We lost this require in the move to the new dependencies class, but maybe we don't need it.
Does oddjobd need to start after sssd? I can't remember why...